src/components/KYC/Form/Controls.tsx

```diff
@@ -65,7 +65,12 @@ export default function Controls({
           Continue
         </button>
       ) : (
-        <button className="btn-orange btn-donate" type="button" onClick={skip}>
+        <button
+          className="btn-orange btn-donate"
+          type="button"
+          onClick={skip}
+          disabled={!hasAgreedToTerms}
```

Comment:
@abbasali1994, @SovereignAndrey 

there's already existing check for this - which button is `enabled` at first which submitting without clicking the checkbox would make it focus there
<img width="709" alt="image" src="https://user-images.githubusercontent.com/89639563/220322400-fe60cc32-8e32-4437-ba62-63cc301ed9c1.png">


I believe this is better UX than button `disabled` at first and user has to figure 
if need to fill-out form or not 
<img width="661" alt="Screenshot 2023-02-21 at 6 40 21 PM" src="https://user-images.githubusercontent.com/89639563/220323053-ff6a5405-6ff2-4d6e-956a-c7a3e93fd8a1.png">


if user starts filling-out form and the endowment doesn't require KYC, the form would sense that user wants to provide KYC anyway so it will make all the fields required
<img width="576" alt="Screenshot 2023-02-21 at 6 44 04 PM" src="https://user-images.githubusercontent.com/89639563/220323600-feee8aa3-19a0-453a-a101-a786b0acfdf1.png">


 which could be prevented if 
 * user just clicks `submit`
 * if KYC not required, get focused on `check-box`
 * if KYC required, gets prompted with required fields

Reply:
@ap-justin The current logic on the site is not what reflective of the latest requirements and real use has shown there are issues with it. Users have reported back that the existing logic results in them missing the T&C box and just clicking submit. This ticket will help cue users in to the fact that they need to do _something_ in order to move forward by having the submit button disabled to start.